addons/account/models/chart_template.py

```diff
@@ -65,7 +65,7 @@ def _avoid_name_conflict(company, template):
                 for index, conflict_taxes in enumerate(conflict_taxes):
                     conflict_taxes.name = f"[old{index if index > 0 else ''}] {conflict_taxes.name}"
 
-        templates_to_create = env['account.tax.template']
+        templates_to_create = env['account.tax.template'].with_context(active_test=False)
```

Comment:
Not really related to what you're fixing, is it ? You're basically making sure inactive templates also get updated (which is correct); shouldn't it be done in a distinct commit, with a dedicated test ?

Reply:
It is actually: inactive templates were added to the recordset, only their `children_tax_ids` were ignored.
But I agree I should add tests for inactive children and templates.